lib/active_merchant/billing/gateways/checkout_v2.rb

```diff
@@ -128,7 +128,7 @@ def add_customer_data(post, options)
       def add_transaction_data(post, options = {})
         post[:payment_type] = 'Regular' if options[:transaction_indicator] == 1
         post[:payment_type] = 'Recurring' if options[:transaction_indicator] == 2
-        post[:payment_type] = 'MOTO' if options[:transaction_indicator] == 3
+        post[:payment_type] = 'MOTO' if options[:transaction_indicator] == 3 || options[:moto]
```

Comment:
Right now I think we only set moto in the stripe interactor. Looking at the code in core and cardserver, I think we should instead use `manual_entry` in `metadata`, this is set to true when it is a manual (draft) order.

Reply:
` we only set moto in the stripe interactor` could we set it in the base interactor instead?

Reply:
either way it should be consistent across all gateways IMO

Reply:
We could but that's what `manual_entry` is for IMHO